storage.bs

```diff
@@ -207,10 +208,10 @@ anticipated that some APIs will be applicable to both <a>storage types</a> going
 <h3 id=storage-keys>Storage keys</h3>
 
 <p>A <dfn export>storage key</dfn> is a <a>tuple</a> consisting of an
-<dfn for="storage key">origin</dfn> (an <a for=/>origin</a>). [[!HTML]]
-
-<p class=XXX>This is expected to change; see
-<a href="https://privacycg.github.io/storage-partitioning/">Client-Side Storage Partitioning</a>.
+<dfn for="storage key">origin</dfn> (an <a for=/>origin</a>), a 
+<dfn for="storage key">top-level site</dfn> (a <a for=/>site</a>), and a
+<dfn for="storage key">cross site ancestry</dfn>
+(a <a for=environment>cross site ancestry</a>). [[!HTML]]
```

Comment:
This is a member, not a type. While some specifications confuse the two (and I've certainly made that mistake in the past), we shouldn't do that for anything new.

Reply:
"cross site ancestry" is the wrong one. What you have here belongs to environment. If that's just a boolean as I'm assuming you could just write "(a boolean)" instead. It should probably also be spelled "cross-site ancestry" with a hyphen.

Reply:
Ah! Sorry for the confusion - "cross site ancestry" is referencing @bvandersloot-mozilla's PR https://github.com/whatwg/html/pull/11133, where he adds the algorithm for calculating cross-site ancestry and allows it to be obtained from the environment.

Reply:
Yes, but he defines a member and you're using it as a type here. The type is a boolean.

Reply:
Alright, I took another swing at it - let me know if it looks better - thank you!